src/DonationSummary/resources/js/summary.js

```diff
@@ -205,12 +205,15 @@ window.GiveDonationSummary = {
      *
      * @param {string/number} amount
      * @param {jQuery} $form
+     * @param {boolean} normalize
      */
-    format_amount: function (amount, $form) {
+    format_amount: function (amount, $form, normalize = true) {
         // Normalize amounts to JS number format
-        amount = amount
-            .replace(Give.form.fn.getInfo('thousands_separator', $form), '')
-            .replace(Give.form.fn.getInfo('decimal_separator', $form), '.');
+        if (normalize) {
+            amount = amount
+                .replace(Give.form.fn.getInfo('thousands_separator', $form), '')
+                .replace(Give.form.fn.getInfo('decimal_separator', $form), '.');
```

Comment:
@glaubersilva Can you confirm that this logic is compatible with the currency that has `,` as decimal separator like `EUR`

Reply:
I did a double check and could notice that the "Donation Total" amount was getting wrong when the value was higher than 999, so I update the normalize amounts logic and removed the extra param added previously.